.vscode/extensions.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Could we add the vs-code stuff to `gitignore` if it is nothing special for this project?

Reply:
Don't know about this stuff... 
@klein0r should answer ;-)

Reply:
seems like vscode needs explicit mapping for the json schema so okay to keep it in for me.